iset.mm

```diff
@@ -129911,11 +129911,18 @@ then the Limited Principle of Omniscience (LPO) implies excluded middle.
     ctiunct.a $e |- ( ph -> F : _om -onto-> ( A |_| 1o ) ) $.
     ctiunct.b $e |- ( ( ph /\ x e. A )
       -> G : _om -onto-> ( B |_| 1o ) ) $.
-    $( The union of countably many countable sets is countable.  The
-       construction here is constructive in the sense that the hypothesis
-       provides a specific class for ` G ` in terms of ` x ` , as opposed to a
-       construction of the form ` A. x e. A E. g ` .  Countable choice would be
-       needed to turn the latter into the former.
+    $( A sequence of enumerations gives an enumeration of the union.  We refer
+       to "sequence of enumerations" rather than "countably many countable
+       sets" because the hypothesis provides more than countability for each
+       ` B ( x ) ` : it refers to ` B ( x ) ` together with the ` G ( x ) `
+       which enumerates it.
+
+       The "countably many countable sets" version could be expressed as
+       ` ( ph /\ x e. A ) -> E. g g : _om -onto-> ( B |_| 1o ) ` and countable
+       choice would be needed to derive the current hypothesis from that.
+
+       Compare with the case of two sets instead of countably many, as seen at
+       ~ unct , in which case we can express countability using ` E. ` .
```

Comment:
```suggestion
       ~ unct , in which case we express countability using ` E. ` .
```

This is merely a nitpick. It may be clearer to explicitly say that countability is the existence of an enumeration

Reply:
The word "can" was intended to emphasize that it isn't possible in other cases, but :shrug: suggestion accepted.